### PR TITLE
links updated in required file

### LINKED ISSUE #9: add react-ui github repository link as path for `edit on github` in documentation website
The file below contains the path 

https://github.com/OSCA-Kampala-Chapter/react-ui/blob/9f0babe7cb83754d1d2e6080e0096ec48a25fa57/reactui-docs/docusaurus.config.js#L42

https://github.com/OSCA-Kampala-Chapter/react-ui/blob/9f0babe7cb83754d1d2e6080e0096ec48a25fa57/reactui-docs/docusaurus.config.js#L49

https://github.com/OSCA-Kampala-Chapter/react-ui/blob/9f0babe7cb83754d1d2e6080e0096ec48a25fa57/reactui-docs/docusaurus.config.js#L122

### PR DESCRIPTION
**Pull request by maddy020**
3 links are updated 
Closes #9

### CODE REVIEW

File: reactui-docs/docusaurus.config.js [hunk 1]
```diff
@@ -39,14 +39,14 @@ const config = {
           // Please change this to your repo.
```
Comment (HassanBahati) [inline reactui-docs/docusaurus.config.js:39]: @maddy020 could we please remove the comments on line 39,40, 46, and 47

Comment (HassanBahati) [reply reactui-docs/docusaurus.config.js:39]: the comments seem unnecessary since we have added the proper links

**Comment by maddy020** [comment]
Changes are done successfully @HassanBahati 
Please review

**Review by HassanBahati — APPROVED** [review]
Looks good to me

Thank you @maddy020

**Pull request merged**